tests/Model/ModelManagerTest.php

```diff
@@ -143,6 +143,12 @@ public function testInstantiateWithDeprecatedRegistryInterface(): void
         $this->assertSame($em, $this->modelManager->getEntityManager('x'));
     }
 
+    /**
+     * NEXT_MAJOR: Remove this test.
+     *
+     * @group legacy
+     * @expectedDeprecation Method Sonata\DoctrineORMAdminBundle\Model\ModelManager::getSortParameters() is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.
```

Comment:
The expectedDeprecation annotation was deprecated in favor of the method inside the test: https://symfony.com/doc/current/components/phpunit_bridge.html#id1

Reply:
That's what we're using in every project currently.
Shouldn't we keep consistency ?

That won't be a real issue since the deprecation is removed in Sonata 4.0

If we start now to move every `@expectedDeprecation`, we'll end to a lot of conflict in master.

Reply:
> If we start now to move every `@expectedDeprecation`, we'll end to a lot of conflict in master.

I haven't said anything about that. IMHO if we are adding new code, I'd rather add non-deprecated one (if it's possible).

Reply:
I think, for the new deprecations we should follow the new format, because we have to get used to it, and we dont know when we will have new release. For older deprecation I would say to not migrate anything